contributing/code/bugs.rst

```diff
@@ -32,9 +33,19 @@ If your problem definitely looks like a bug, report it using the official bug
 * Give as much detail as possible about your environment (OS, PHP version,
   Symfony version, enabled extensions, ...);
 
+* If you want to provide a stack trace you got on an html page, be sure to
```

Comment:
If you expose sensitive information you are already F**ked, posting this in a screenshot will not prevent anyone from abusing this information. Not to mention that search engines can perform OCR.